doc/index.rst

```diff
@@ -6,38 +6,65 @@
 lsst.ts.ATDome
 ##############
 
-Controller for the LSST auxiliary telescope dome.
+.. image:: https://img.shields.io/badge/SAL\ Interface-green.svg
+    :target: https://ts-xml.lsst.io/sal_interfaces/ATDome.html
+.. image:: https://img.shields.io/badge/GitHub-green.svg
+    :target: https://github.com/lsst-ts/ts_ATDome
+.. image:: https://img.shields.io/badge/Jira-green.svg
+    :target: https://jira.lsstcorp.org/issues/?jql=labels+%3D+ts_ATDome
 
-.. _lsst.ts.ATDome-using:
+.. _Overview:
 
-Using lsst.ts.ATDome
-====================
+Overview
+========
 
-The primary classes are:
+Controller for the Vera Rubin Observatory Auxiliary Telescope dome.
 
-* `ATDomeCsc`: controller for the auxiliary telescope dome.
-* `MockDomeController`: simulator for the auxiliary telescope dome TCP/IP interface.
+.. _lsst.ts.ATDome.usage:
 
-Run the ``ATDome`` controller  using ``bin/run_atdome.py`` (which only exists after you build the package).
+Usage
+=====
 
-.. _building single package docs: https://developer.lsst.io/stack/building-single-package-docs.html
+Start the ATDome CSC as follows:
 
-.. _lsst.ts.ATDome-contributing:
+.. prompt:: bash
 
-Contributing
-============
+    run_atdome.py
 
-``lsst.ts.ATDome`` is developed at https://github.com/lsst-ts/ts_ATDome.
-You can find Jira issues for this module using `labels=ts_ATDome <https://jira.lsstcorp.org/issues/?jql=project%20%3D%20DM%20AND%20labels%20%20%3D%20ts_ATDome>`_.
+For on-sky observing, dome azimuth is automatically commanded to follow the telescope by `ts_ATDomeTrajectory <https://ts-atdometrajectory.lsst.io/>`_.
+If you do not want dome azimuth to follow the telescope then command ATDomeTrajectory to go to the DISABLED state.
+See ATDome `SAL interface <https://ts-xml.lsst.io/sal_interfaces/ATDome.html>`_ for commands, events and telemetry.
 
-.. _lsst.ts.ATDome-pyapi:
+Stop the CSC by commanding it to go to the OFFLINE state.
 
-Python API reference
-====================
+.. _lsst.ts.ATDome.configuration:
 
-.. automodapi:: lsst.ts.ATDome
-   :no-main-docstr:
-   :no-inheritance-diagram:
+Configuration
```

Comment:
My intent was to make the Configuration and Simulator sections part of the user's guide. I think it is the kind of information that a user will want to know. Notice that Configuration and Simulation do appear in the table of contents so they are still easy to find. I'm certainly willing to discuss this.

Reply:
So configuration is certainly something that the user and developer need to know, which is why it had it's own section on the front page. 
I suppose I don't see why a standard user needs to know about the simulator? I'd of thought that's a very highly advanced use-case which is used by developers (or in the case of the NTS stoodup by devs for users).

Reply:
I was thinking of these roles:
* User: somebody who starts, stops, or commands the CSC.
* Developer: somebody who modifies the code or needs to read it for some other reason.
I feel this makes a very natural division. Clearly there can be different kinds of users, but I worry it will difficult to divide the document between people who only want to command the CSC and those who want to do something a bit deeper, such as change its configuration, start or stop it, or run in simulation mode. All those things except starting a CSC are done by commanding it.